### PR TITLE
define recursive attribute and clarify level attribute

### PR DESCRIPTION
**Pull request by dericed**

### CODE REVIEW

File: specification.markdown [hunk 1]
```diff
@@ -223,7 +223,7 @@ Within an EBML Schema the `<element>` uses the following attributes to define an
 | attribute name | required | definition |
 |----------------|----------|------------|
 | name           | Yes      | The official human-readable name of the EBML Element. The value of the name MUST be in the form of an NCName as defined by the [XML Schema specification](http://www.w3.org/TR/1999/REC-xml-names-19990114/#ns-decl). |
-| level          | Yes      | The level notes at what hierarchical depth the EBML Element may occur within an EBML Document. The Root Element of an EBML Document is at level 0 and the Elements that it may contain are at level 1. The level MUST be expressed as an integer; however, the integer may be followed by a '+' symbol to indicate that the EBML Element is valid at any higher level.  |
+| level          | Yes      | The level notes at what hierarchical depth the EBML Element may occur within an EBML Document. The Root Element of an EBML Document is at level 0 and the Elements that it may contain are at level 1. The level MUST be expressed as an integer. Note that for Elements defined as `global` and `recursive` the Element MAY occur at a level greater than or equal to the defined `level`.|
```
Comment (mbunkus) [inline specification.markdown]: That sentence seems to be a bit too complex for no particular reason. What about the slightly shorter "Note that Elements defined as `global` and `recursive` MAY occur…"? Then again I'm not a native speaker and may be off base here :)

File: specification.markdown [hunk 2]
```diff
@@ -232,6 +232,7 @@ Within an EBML Schema the `<element>` uses the following attributes to define an
...
 | type           | Yes      | As defined within the [section on EBML Element Types](#ebml-element-types), the type MUST be set to one of the following values: 'integer' (signed integer), 'uinteger' (unsigned integer), 'float', 'string', 'date', 'utf-8', 'master', or 'binary'. |
 | unknownsizeallowed | No       | A boolean to express if an EBML Element MAY be used as an `Unknown-Sized Element` (having all VINT\_DATA bits of Element Data Size set to 1). The `unknownsizeallowed` attribute only applies to Master-elements. If the `unknownsizeallowed` attribute is not used it is assumed that the element is not allowed to use an unknown Element Data Size. |
+| recusrive | No       | A boolean to express if an EBML Element MAY be stored recursively. In this case the Element MAY be stored at levels greater that defined in the `level` attribute if the Element is a Child Element of a Parent Element with the same Element ID. The `recursive` attribute only applies to Master-elements. If the `recursive` attribute is not used it is assumed that the element is not allowed to be used recursively.|
```
Comment (mbunkus) [inline specification.markdown]: Spelling mistake in the first word: "recusrive" → "recursive"

Comment (dericed) [inline specification.markdown]: fixed, rebased, and force pushed

**Comment by mbunkus** [comment]
Thanks :)

**Pull request merged**